### PR TITLE
Makes the vent clog event happen more often

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Dax Dupont
tweak: Vent clog events are more likely to happen.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

I said in the original PR where I rebalanced vent clogs that I would eventually evaluate the probability of the events.

Since they don't have a too large impact, I'm rejiggling some of the requirements and time required.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
? These things are instant round enders

**Comment by DaxDupont** [comment]
@PKPenguin321 since when, these aren't the goof ones since a long time?

They do barely anything. 

90% of the people get hit with things that don't really affect them much on CATASTROPHIC.

On the lower ends it's way way less. It's not like gooffoam where half the station got TB and the other was on fire.

**Comment by zxaber** [comment]
This is just when it floods the station with various shit somehow magically even if the scrubber is disconnected, right?

It's not usually round-ending, but it's not terribly interesting either. You just pop your gas mask on and stand still for 10 seconds. There's no real gameplay value other than just spontaneous waiting. Then the station is covered in random glitter that does nothing and everyone ignores.

It's sorta like hygiene; it's not really positive or negative, it's just a random but short timesink that does almost nil. _Maybe_ your room floods with acid and so you take the chip damage as you slide to another room and that's the end of it.

**Comment by DaxDupont** [comment]
you've just more or less described most events which is fine.

I just want to see more variety in events, 90% of the time it's just electric storms or grid check

**Comment by wesoda25** [comment]
It happens more than enough currently. No point to boost the probability.

**Comment by DaxDupont** [comment]
@wesoda25 It doesn't. 

Especially on servers with low pop. Also this doesn't change the WEIGHT of the event. It might just happen EARLIER with LESS players.

Though the less players won't affect bagil most of the time since it's already above it for most of em.

**Comment by DaxDupont** [comment]
tl;dr it can happen on low pop now instead of 25+ players only

**Comment by PKPenguin321** [comment]
all i know is every time they happen half of the halls are filled with plasma and catch fire

**Comment by DaxDupont** [comment]
Most of the time I see them they just color rooms and some people get drunk or high

**Comment by zxaber** [comment]
I am almost certain this even can't spawn plasma. It _can_ spawn pink glitter, though.

**Comment by subject217** [comment]
threatening and catastrophic both have rather concerning reagents in them and dax is making both of those more common by decreasing min players and earliest possible time. if this pr was just about clogged vents: normal you might have a point but it's not.

**Comment by WaterStBlues** [comment]
I don't see why this is a problem lmao, this shit is more often funny than harmful. merge

**Comment by DaxDupont** [comment]
they both still have tiny weights which makes them /very/ rare. The weight on catastrophic is 2.

**Comment by DaxDupont** [comment]
I'll undo the minutes change, 5 minutes aint gonna change much anyways but it'll satisfy the crybabies at least a little.

**Review by Cyberboss — APPROVED** [review]

**Comment by DaxDupont** [comment]
badabing badaboom
